src/main/scala/net/snowflake/spark/snowflake/SnowflakeJDBCWrapper.scala

```diff
@@ -715,9 +708,9 @@ private[snowflake] class SnowflakeSQLStatement(
     bindVariableEnabled: Boolean
   )(implicit conn: Connection): ResultSet =
     if (bindVariableEnabled) executeWithBindVariable(conn)
-    else executeWithoutBindVaribale(conn)
+    else executeWithoutBindVariable(conn)
```

Comment:
I don't see any difference for line 711/713. Why they are diff?

Reply:
I fixed a typo. "Varibale" vs "Variable".

Reply:
Good catch.